lib/payoneer/client.rb

```diff
@@ -39,14 +39,60 @@ def payout(program_id:, payment_id:, payee_id:, amount:, description:, payment_d
       )
     end
 
+    # Includes additional items as needed to be Payoneer SAFE compliant
+    def expanded_payout(payee_id:, client_reference_id:, amount:, currency: 'USD', description:, payout_date: Time.now, seller_id:, seller_name:, seller_url:, seller_type: 'ECOMMERCE', path:, credentials_type:, token: '', user_name: '', password: '')
```

Comment:
Don't default `currency`, let that be explicit.

Reply:
The normal payout defaults to USD, should I change it there too?